scipy/spatial/tests/test_spherical_voronoi.py

```diff
@@ -170,3 +172,15 @@ def test_single_hemisphere_handling(self):
             dots = np.einsum('ij,ij->i', sv.vertices, triangles[:, 0])
             circumradii = np.arccos(np.clip(dots, -1, 1))
             assert np.max(circumradii) > np.pi / 2
+
+    def test_minimum_dimensionality(self):
+        with assert_raises(ValueError):
+            sv = spherical_voronoi.SphericalVoronoi(np.linspace(0, 1, 10))
+
+    @pytest.mark.parametrize("dim", range(2, 6))
+    def test_higher_dimensions(self, dim):
+        n = 100
+        points = np.random.randn(n, dim)
+        points /= np.linalg.norm(points, axis=1)[:, np.newaxis]
+        sv = SphericalVoronoi(points)
+        assert sv.vertices.shape[1] == dim
```

Comment:
This is a sensible check for higher dims. I'm having a hard time thinking of easy-to-understand ways to write unit tests for probing higher-dimensional spherical Voronoi results, especially given the complications for plotting > 3D to reason about what is going on.

One perhaps obvious idea is to have coordinates with i.e., [x, y, z, b], where all b values are constant--so that the 4th dimension is predictably not doing much & then compare back to a 3D result that is well understood.

Reply:
I have added two unit tests for the cross-polytope https://en.wikipedia.org/wiki/Cross-polytope
This is a simple test for higher-dimensional input.

It doesn't catch everything but it is a good sanity check on basic behaviour.

Reply:
Here, can you also test than `len(sv.regions) == n`?

Reply:
Sorry - I overlooked this.  I have added the test.